File: Two_Pointers/Patterns/Pattern1/Pattern1.java
class Pattern1{
    public static void main(String[] args) {
        int[] num={1,5,3,70,2,11,4};
        int sum=num[0]+num[1]+num[2];
        int max=sum;
        int k=3;
        int left=0;
        int right=k;
        while(right<num.length){
            sum=sum-num[left++]+num[right++];
            max=Math.max(max,sum);
        }
        System.out.println(max);
    }
}

File: Two_Pointers/Patterns/Pattern1/Pattern1.py
lst=[15,1,23,20,58,5,2,14]
k=3
summ=0
for i in range(3):
    summ+=lst[i]
maxx=summ
left=0
right=3

while right<len(lst):
    summ=summ-lst[left]+lst[right]
    maxx=max(summ,maxx)
    left+=1
    right+=1

print(maxx)
